lib/theme.json

```diff
@@ -1,5 +1,8 @@
 {
 	"version": 2,
+	"behaviors": {
+		"lightbox": true
+	},
```

Comment:
What's the reasoning behind using `behaviors.lightbox` instead of `behaviors.blocks.core/image.lightbox`?

Reply:
I thought it unnecessary to nest the `behaviors` under `blocks.core/image`. I think that it's simpler and more readable without the nesting. 

Why was it considered necessary in the first place? As far as I can tell, the only use case for this namespacing would be to provide the `lightbox` behavior (or some other behaviors) as an option for one block but not for another. But we are not planning to add any other blocks with the lightbox behavior at the moment as far as I know?

Reply:
I don't know. Gallery probably. But even if we don't have plans at the moment, there's the possibility. 

`behaviors.lightbox` should be something to control _all_ the lightboxes of _all_ the blocks, not just one block. So it'd be safer to use `behaviors.blocks.core/image.lighbox` for now.